### PR TITLE
CPLAT-6258 Setup Travis CI

### PR DESCRIPTION
**Pull request by evanweible-wf**
Now that the repo is OSS, need to get Travis CI running.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+
+dart:
+  - 2.3.2
+  - stable
+  - dev
```
Comment (maxwellpeterson-wf) [inline .travis.yml]: need all three of these?

Comment (evanweible-wf) [reply .travis.yml]: Maybe we don't need stable.. but 2.3.2 is the package minimum so it prevents regressions

**Comment by evanweible-wf** [comment]
QA +1
- [x] CI passes

@Workiva/release-management-p

**Pull request merged**